dm/syncer/schema.go

```diff
@@ -167,3 +171,63 @@ func (s *Syncer) OperateSchema(ctx context.Context, req *pb.OperateWorkerSchemaR
 	}
 	return "", nil
 }
+
+// listMigrateTargets list all synced schema and table names in tracker.
+func (s *Syncer) listMigrateTargets(req *pb.OperateWorkerSchemaRequest) (string, error) {
```

Comment:
schema tracker is loaded lazily right now, so this api may not return all route relations.
changed to load them all before syncer run in #4770

Reply:
yes, it is as expected, here it is only necessary to return the routing rules for the data that has been synced to downstream